material_studies/rally/lib/tabs/overview.dart

```diff
@@ -111,42 +108,42 @@ class _AlertsView extends StatelessWidget {
 }
 
 class _FinancialView extends StatelessWidget {
-//  _FinancialView({this.title, this.total, this.financialItemViews});
+  _FinancialView({this.title, this.total, this.financialItemViews});
 
-//  final String title;
-//  final double total;
-//  final List<FinancialEntityCategoryView> financialItemViews;
+  final String title;
+  final double total;
+  final List<FinancialEntityCategoryView> financialItemViews;
 
   @override
   Widget build(BuildContext context) {
     final theme = Theme.of(context);
     return Container(
       color: RallyColors.cardBackground,
-//      child: Column(
-//        crossAxisAlignment: CrossAxisAlignment.stretch,
-//        children: <Widget>[
-//          Padding(
-//            padding: EdgeInsets.all(16),
-//            child: Text(title),
-//          ),
-//          Padding(
-//            padding: EdgeInsets.only(left: 16, right: 16),
-//            child: Text(
-//              Formatters.usdWithSign.format(total),
-//              style: theme.textTheme.body2.copyWith(
-//                fontSize: 44.0,
-//                fontWeight: FontWeight.w600,
-//              ),
-//            ),
-//          ),
-//          ...financialItemViews.sublist(0, 3),
-//          FlatButton(
-//            child: Text('SEE ALL'),
-//            textColor: Colors.white,
-//            onPressed: () {},
-//          ),
-//        ],
-//      ),
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.stretch,
+        children: [
+          Padding(
+            padding: EdgeInsets.all(16),
+            child: Text(title),
+          ),
+          Padding(
+            padding: EdgeInsets.only(left: 16, right: 16),
+            child: Text(
+              Formatters.usdWithSign.format(total),
+              style: theme.textTheme.body2.copyWith(
+                fontSize: 44.0,
+                fontWeight: FontWeight.w600,
+              ),
+            ),
+          ),
+          ...financialItemViews.sublist(0, 3),
```

Comment:
You have hardcoded data, which works around the problem, but what would happen here if the list were empty? It would be good to show devs that the widget (which currently takes whatever list it's given) defends against or accounts for that.

Reply:
Done